recipes/pandas/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version="0.18.1" %}
+
+package:
+    name: pandas
+    version: {{ version }}
+
+source:
+    fn: pandas-{{ version }}.tar.gz
+    url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
+    sha256: 1f3136f69a58d490ba9f9874e67601042a11138270515dedcf6979c003bba79b
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --root=/
```

Comment:
Agreed. I have thought a bit about this because there is a bunch of tedious stuff that should be done away with.

Maybe some of this (whitespace) can happen in the conversion process.

Some of it could be captured in some jinja import type thing (setuptools, maybe compiler stuff).

Also, some of it like weird compiler junk should go in some compile meta package that just configures things.

These are a few thoughts I have had.